src/modules/reservations-api.js

```diff
@@ -7,4 +7,18 @@ const fetchShow = async (showId) => {
   return show;
 };
 
-export default fetchShow;
\ No newline at end of file
+const fetchcomments = async () => {
```

Comment:
Please consider receiving a show id so you can make the API call dynamically. Seems like it's just calling to a static hardcoded value of 1 (see next comment)